diff --git a/src/chat/chat.coffee b/src/chat/chat.coffee
index 5bebdf3..b92b33a 100644
--- a/src/chat/chat.coffee
+++ b/src/chat/chat.coffee
@@ -14,7 +14,7 @@ class Chat extends EventEmitter
     @_initializeRemoteConnection()
     @_initializeStorage()
     @_initializeScripts()
-    @_listenForUpdates()
+    @_promptPackageObsolete()
     @_keyboardShortcutMap = new KeyboardShortcutMap
 
     @updateStatus()
@@ -90,6 +90,12 @@ class Chat extends EventEmitter
     chrome.runtime.onUpdateAvailable?.addListener =>
       @_promptToUpdate()
 
+  _promptPackageObsolete: ->
+    message = "This version of CIRC is now obsolete. Please switch to " +
+        "the main version. [chrome store]"
+    @notice.prompt message, =>
+      window.open('https://chrome.google.com/webstore/detail/circ/bebigdkelppomhhjaaianniiifjbgocn');
+
   _promptToUpdate: ->
     message = "A new versionn of CIRC is available. Would you like to " +
         "restart and update? [update]"
